test/PushNotificationTests/LRPTests.cpp

```diff
@@ -179,5 +179,45 @@ namespace Test::PushNotifications
             VERIFY_IS_FALSE(isRunning);
         }
 
+        TEST_METHOD(AddToastRegistrationMappingNoSink)
+        {
+            VERIFY_SUCCEEDED(::CoInitializeEx(nullptr, COINITBASE_MULTITHREADED));
```

Comment:
Do you have a task to move CoInitializeEx/CoUninitialize to the class setup/cleanup tests function?

Reply:
Moving to methodinit/unit in iteration